### PR TITLE
Windows build

### PR DESCRIPTION
**Pull request by dloney**

### CODE REVIEW

**Comment by dloney** [comment]
I don't want to merge this in before the previous pull request since it will also introduce the compiler upgrades.

File: docs/pages/1712/solution/building.html [hunk 1]
```diff
@@ -116,15 +116,39 @@ <h2><a href="#">Building OSTRICH</a></h2>
 							<p align="left">Initial Publication:</p>
 							<time class="time" datetime="2015-11-01">September 16, 2020</time>
 							<p align="left">Modified:</p>
-							<time class="time" datetime="2015-11-01">September 16, 2020</time>
+							<time class="time" datetime="2015-11-01">September 29, 2020</time>
```
Comment (julemai) [inline docs/pages/1712/solution/building.html:119]: Why is `datetime`  Nov 1, 2015 and date given Sep 29, 2020?

File: docs/pages/1712/solution/building.html [hunk 2]
```diff
@@ -214,20 +238,128 @@ <h2>1.3 Serial build with database output </h2>
...
+						add _CRT_SECURE_NO_WARNINGS. This will tell the compiler to default to the C++ standard.
+					</p>
+					
+					<h2 id="sec_vs_troubleshooting">2.5 Troubleshooting</h2>
+					<p align="justify">
+						If you're seeing header missing issues, double check that the configuration you set the compilation options for is the same as the one that is 
+						enabled in the Visual Studio IDE. In general, it's recommended to use x64. Confirm that you set the compilation options for the architecture/release
+						combination you're using within the IDE.
+					</p>
+
```
Comment (julemai) [inline docs/pages/1712/solution/building.html:362]: Above looks good to me but hard to test because I don't use windows. :)

**Review by julemai — APPROVED** [review]
Made a couple of comments. Nothing serious.

File: docs/pages/1712/solution/building.html (hunk 1 above)
Comment (dloney) [reply docs/pages/1712/solution/building.html:119]: I think that's part of the template since it's in the tag.

**Pull request merged**